infrastructure/docker-compose.deploy.yml

```diff
@@ -405,6 +405,8 @@ services:
       - '/opt/opencrvs/infrastructure/monitoring/elastalert/rules:/opt/elastalert/rules'
     networks:
       - overlay_net
+    depends_on:
```

Comment:
The depends_on option is ignored when [deploying a stack in swarm mode](https://docs.docker.com/reference/cli/docker/stack/deploy/) with a version 3 Compose file.

https://docs.docker.com/compose/compose-file/compose-file-v3/#depends_on